docs/eventing/event-delivery.md

```diff
@@ -90,6 +90,32 @@ spec:
     backoffDelay: <ISO8601 duration>
 ```
 
+Failed events may, depending on the specific Channel implementation in use, be
+enhanced with extension attributes prior to forwarding to the`deadLetterSink`.
+These extension attributes are as follows...
+
+- **knativeerrorcode**
+    - **Type:** Int
+    - **Description:** The HTTP Response **StatusCode** from the final event
+      dispatch attempt.
+    - **Constraints:** Should always be present as every HTTP Response contains
+      a **StatusCode**.
+    - **Examples:**
+        - "500"
+        - ...any HTTP StatusCode...
+
+- **knativeerrordata**
+    - **Type:** String
+    - **Description:** The HTTP Response **Body** from the final event dispatch
+      attempt.
+    - **Constraints:** Will not be present if the HTTP Response **Body** was
+      empty. Will be truncated to a maximum length of 1024 bytes which could
+      render JSON payloads invalid / incomplete.
```

Comment:
Keep it vague the maximum length truncation, so we can eventually modify the length in future and people won't depend on it... Something like:

> The original Response body might be truncated

Reply:
👍